src/stream/src/executor/hop_window.rs

```diff
@@ -87,6 +87,18 @@ impl HopWindowExecutor {
             output_indices,
             ..
         } = *self;
+
+        if !window_size.is_positive() || !window_slide.is_positive() {
+            return Err(ExprError::InvalidParam {
+                name: "window",
+                reason: format!(
+                    "window_size {} and window_slide {} must be positive",
+                    window_size, window_slide
+                ),
+            }
+            .into());
+        }
```

Comment:
Is it duplicate with the param check in planner?

Reply:
This is intentional, my thinking is that in the future if `window_size` / `window_slide` can be columns, this will only be able to be caught during runtime (i.e. in the executor) and not during planner phase.. Hence I added a check here as well.

If both `window_size` and `window_slide` are always literals, this check can be removed.

Reply:
It doesn't make sense for window_size / window_slide to be dynamic. Will remove this check.

Reply:
Yeah we have validated that they must be literals.